kedro-org/kedro-viz
# Update package dependency version to work with Node v16

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

We need to update a dependency to work with Node v16.

## QA notes

Checkout the branch, delete your `node_modules` folder and your `package-lock.json` file, make sure you're using Node v16, and then run `npm install`. Everything should work fine.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1037"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
